apps/condo/domains/common/components/settings/SettingsPageContent.tsx

```diff
@@ -31,33 +21,18 @@ export const SettingsPageContent: React.FC<PageContentProps> = ({ settingsTabs,
         return router.push(newRoute)
     }, [router])
 
-    const settingsTabPanes = useMemo(() => settingsTabs.map(tab => (
-        <Tabs.TabPane
-            key={tab.key}
-            tab={<Tab title={tab.title} eventName={tab.eventName}/>}
-            {...tab}
-        >
-            {tab.content}
-        </Tabs.TabPane>
-    )), [settingsTabs])
-
-    const { breakpoints } = useLayoutContext()
-
-    const SettingsTabsComponent = !breakpoints.DESKTOP_SMALL ? TopRowTabs : SideBlockTabs
-
-    return settingsTabs.length === 1 ? (
-        settingsTabs[0].content
-    ) : (
-        <SettingsTabsComponent
-            tabPosition={!breakpoints.DESKTOP_SMALL ? 'top' : 'right'}
-            type='card'
-            defaultActiveKey={defaultTab}
-            activeKey={defaultTab}
-            tabBarGutter={8}
-            style={SETTINGS_TABS_STYLES}
-            onChange={handleTabChange}
-        >
-            {settingsTabPanes}
-        </SettingsTabsComponent>
-    )
+    return <>
+        {
+            settingsTabs.length === 1 ? (
```

Comment:
Is it a case now? We've already have 3 mandatory tabs for all employees :)

Reply:
i think now it's redundant, remove it